vignettes/strayr.Rmd

```diff
@@ -0,0 +1,154 @@
+---
+title: "StrayR"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{StrayR}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+
+## Under Construction
+
+
+## Using ABS structures
+
+Loading the package will lazily load the structures listed above. Call
+them with their name:
+
+```{r actual-load, include=FALSE}
+# use this to make sure README runs with current build
+# rather than installed version
+devtools::load_all()
+library(dplyr, warn.conflicts = FALSE)
+```
+
+```{r fake-load, eval=FALSE}
+library(strayr)
+#> Loading required package: absmapsdata
+library(dplyr)
+```
+
+```{r glimpse}
+glimpse(anzsco)
+glimpse(anzsic)
+glimpse(asced_foe)
+glimpse(asced_qual)
+glimpse(sa42016)
+glimpse(ced2018)
+```
+
+
+## Converting state names and abbreviations
+
+The `clean_state()` function makes it easy to wrangle vectors of State names and abbreviations - which might be in different forms and possibly
+misspelled.
+
+Let's start with a character vector that includes some misspelled State names,
+some correctly spelled state names, as well as some abbreviations both malformed
+and correctly formed.
+
+```{r create-string}
+
+x <- c("western Straya", "w. A ", "new soth wailes", "SA", "tazz", "Victoria",
+       "northn territy")
```

Comment:
There's some errant empty lines before some of the examples here that might look tidier if removed (I think they were in the original README)
```suggestion
x <- c("western Straya", "w. A ", "new soth wailes", "SA", "tazz", "Victoria",
       "northn territy")
```